data/static/base_info.csv

```diff
@@ -1,4 +1,5 @@
 name_en,first_published,org_type,registry_id,baseline
+Agence Française de Développement,30-09-2016,Other Public Sector,afd,0
```

Comment:
The date should be in the format `yyyy-mm-dd`

Reply:
Thanks! Just to say we are also holding off adding AFD for now.

Reply:
Date formatting fixed in 9f5a055 (although I have received notification from @PetyaKangalova that AFD will be removed from the Grand Bargain signatories).